### DISCUSSION TITLE
How setting class on condition to concatenate it with other classes?

### DISCUSSION THREAD

**Discussion by sergeynilov** [alpinejs/alpine#683] [4. General]
Hello,
Reading doc
https://github.com/alpinejs/alpine
I did not find how setting class on condition to concatenate it with other classes?
I tried :
```
  <button :class="'btn btn-outline-secondary '+{ 'active': tab === 'extra-details' } " @click.prevent="tab = 'extra-details'">
    <h3>Extra details</h3>

```
But looks like that does not work...

alpine@v2.x.x
Thanks!

**Comment by SimoTod** [comment]
Yeah, you can't concatenate a string to an object.
Any of the following should do. Depending on you use case, one could be more suitable of the othes.
- `<button class="btn btn-outline-secondary" :class="{ 'active': tab === 'extra-details' }">`
- `<button :class="(tab === 'extra-details') ? 'btn btn-outline-secondary active' : 'btn btn-outline-secondary'">`
- `<button :class="{ 'active': tab === 'extra-details', 'btn btn-outline-secondary': true }">`
